setup.py

```diff
@@ -57,7 +57,7 @@ def is_requirement(line: str) -> bool:
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
     python_requires=">=3.8",
-    install_requires=load_requirements("base.in"),
+    install_requires=load_requirements("base.txt"),
```

Comment:
It is generally not recommended to pin requirements in Python projects. This is because when a user runs `pip install tutor someotherreq`, installation will fail if the requirements are not exactly the same. I can't find the Python docs reference page anymore, though.